hypothesis-python/src/hypothesis/internal/conjecture/junkdrawer.py

```diff
@@ -45,25 +47,25 @@ def array_or_list(
 
 
 def replace_all(
-    buffer: Sequence[int],
-    replacements: Iterable[Tuple[int, int, Sequence[int]]],
-) -> bytes:
-    """Substitute multiple replacement values into a buffer.
+    l: Sequence[T],
```

Comment:
```suggestion
    ls: Sequence[T],
```
`l` alone is easy to confuse for `1` or `|`, in many fonts.